api/Navigator.json

```diff
@@ -534,6 +534,54 @@
           }
         }
       },
+      "contacts": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/contacts",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": "80"
```

Comment:
Confirmed supported on Chrome for Android and *not* macOS using https://mdn-bcd-collector.appspot.com/tests/api/Navigator/contacts

Moreover, Chrome 80 is confirmed by https://storage.googleapis.com/chromium-find-releases-static/967.html#967ef324a2761a4615713e3fe152423d1715d7f6